src/administration/entities/project-administration-scoped.entity.ts

```diff
@@ -52,6 +52,11 @@ export const projectAdministrationAccessFilter = async (
   if (!projectPrincipal) return { _id: null };
 
   switch (type) {
+    case 'update':
+      return {
+        createdBy: projectPrincipal.id,
+        project: projectPrincipal.project
+      };
```

Comment:
Not safe a safe condition for the update filter, rather disable the filter during `nativeUpdate`